php_phongo.c

```diff
@@ -1162,6 +1173,35 @@ php_phongo_server_description_type_t php_phongo_server_description_type(mongoc_s
 	return PHONGO_SERVER_UNKNOWN;
 }
 
+bool php_phongo_server_description_to_zval(zval* retval, mongoc_server_description_t* sd) /* {{{ */
```

Comment:
This is no longer used. I'd suggest removing this from the current PR and keeping a copy of it for the TopologyDescription PR if you decide to reintroduce it.

I realize that may sound futile, but it will keep the commits more focused on the necessary changes if/when we need to refer back to them.